### PR TITLE
fix(install-chrome.sh): Chrome install not working on ARM executors

### PR DESCRIPTION
**Pull request by itsyuvalcohen**
### Checklist

- [x] All new jobs, commands, executors, parameters have descriptions
- [x] Examples have been added for any significant new features
- [x] README has been updated, if necessary

### Motivation, issues

Chrome installations on ARM executors fails due to a bad architecture check causing it to use amd64 instead of arm64

### Description

Fixed ENV_IS_ARM variable

### CODE REVIEW

File: src/scripts/install-chrome.sh [hunk 1]
```diff
@@ -106,9 +106,9 @@ else
   # download chrome
   echo "Preparing Chrome installation for Debian-based systems"
   if [[ "$ORB_PARAM_CHROME_VERSION" == "latest" ]]; then
-    ENV_IS_ARM=$(! dpkg --print-architecture | grep -q arm; echo $?)
+    ENV_IS_ARM=$(dpkg --print-architecture | grep arm64)
```
Comment (ryanbourdais) [inline src/scripts/install-chrome.sh:109]: I just tried test running these changes and now most of the tests are failing, I think that this line should go back to how it was originally but replacing line 111 with `if [ "$ENV_IS_ARM" == "1" ]; then` would fix this issue

**Review by ryanbourdais — CHANGES_REQUESTED** [review]
Thanks for the contribution, I left a note in your changes

**Comment by itsyuvalcohen** [comment]
> Thanks for the contribution, I left a note in your changes

I figured out google doesn't even publish google-chrome-stable for arm64, I think we should exit with an error saying it isn't supported

**Comment by ryanbourdais** [comment]
> > Thanks for the contribution, I left a note in your changes
> 
> I figured out google doesn't even publish google-chrome-stable for arm64, I think we should exit with an error saying it isn't supported

If you would like to make that change on this PR I can get that merged in, otherwise I can just close this PR out.

**Review by ryanbourdais — APPROVED** [review]
Looks good to me

**Comment by m3thom** [comment]
If I understand correctly, we should be able to install Chrome on ARM when this issue is resolved, no?
https://github.com/GoogleChromeLabs/chrome-for-testing/issues/1

**Pull request merged**